src/node_kernels/TKESSTIDDESNodeKernel.C

```diff
@@ -49,7 +49,7 @@ TKESSTIDDESNodeKernel::setup(Realm& realm)
   fOneBlend_ = fieldMgr.get_field<double>(fOneBlendID_);
   ransIndicator_ = fieldMgr.get_field<double>(ransIndicatorID_);
   // call modify before this field gets modified in kernel execute phase
-  ransIndicator_.modify_on_device();
+  ransIndicator_.sync_to_device();
```

Comment:
is there a modify_on_device for this field somewhere else?

Reply:
Actually looking at this again. this is probably an unnecessary call here

Reply:
Deleted it